### ISSUE TITLE
Upcoming v0.64.3 cherrypicks discussion 

### ISSUE THREAD

**Issue by kelset** [react-native-community/releases#232]
Conversations on this thread are limited to 0.64 releases major issues and backport (cherry-pick) requests from commits that are already on master.

An example of a good such request is a bug fix for a serious issue that has been merged into master but did not make the 0.64.2 cut, with a link to the specific commit hash on master with the commit to cherry-pick, like this example link: https://github.com/facebook/react-native/commit/bd2b7d6c0366b5f19de56b71cb706a0af4b0be43

In other words, if you cannot point to a particular commit on master, then your request likely belongs as a new issue in http://github.com/facebook/react-native/issues.

**Comment by omnikron** [comment]
Would it be possible to include https://github.com/facebook/react-native/commit/bdfe2a5 , which fixes an issue with broken iOS builds (https://github.com/facebook/react-native/issues/31193)? The commit is in master since 16th April but we are facing the same issue today after upgrading to 0.64.2.  We added patch-package to apply the commit on top of 0.64.0 every time we build our app and can confirm that it works.

**Comment by danilobuerger** [comment]
@omnikron @kelset wasn't this already requested to be cherry picked in 0.64.2?

**Comment by kelset** [comment]
We will also cherry pick this: https://github.com/facebook/react-native/pull/31682

re: this commit https://github.com/facebook/react-native/commit/bdfe2a5 as pointed out couldn't get cherry picked cleanly, so it's not bound to happen as is. What would need to happen is to have a dedicated PR that targets 0.64 and replicates the changes needed in a way that goes in cleanly.

**Comment by danilobuerger** [comment]
I have it as a commit in my fork that cleanly cherry-picks onto 0.64.2: https://github.com/feastr/react-native/commit/d9f7770e9d26ac29f0af9178b49a08aaba7584a2 if anybody wants to PR it to the 0.64 branch, feel free.

**Comment by omnikron** [comment]
> I have it as a commit in my fork that cleanly cherry-picks onto 0.64.2: [feastr/react-native@d9f7770](https://github.com/feastr/react-native/commit/d9f7770e9d26ac29f0af9178b49a08aaba7584a2) if anybody wants to PR it to the 0.64 branch, feel free.

I've done just that: https://github.com/facebook/react-native/pull/31688

@kelset is this sufficient for cherry-picking? Happy to help in any other way.

**Comment by kelset** [comment]
We also want https://github.com/facebook/react-native/commit/4d40b53c12c8ad52760c63cacde417ee876bdfb1 IIRC

**Comment by andreialecu** [comment]
I hope this is not dumb to ask for here, but updating to 0.65 will be a hard task for a lot of people, and this small fix would be incredible to have as part of 0.64.

Font weights have been broken on Android since forever, and this finally fixes them. It's been merged recently:

https://github.com/facebook/react-native/pull/29117

**Comment by birkir** [comment]
Maybe we can include accessible colors for DynamicColorIOS in 64.3? facebook/react-native@4b9d9dd

**Comment by Attimo91** [comment]
I think that this is very important: https://github.com/facebook/react-native/commit/70da640

**Comment by mikehardy** [comment]
@Attimo91 JCenter is actually not that important right now: 

https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/

> UPDATE 4/27/2021: We listened to the community and will keep JCenter as a read-only repository indefinitely. Our customers and the community can continue to rely on JCenter as a reliable mirror for Java packages.

...slowly exhale on that one. It should be removed, but zero need for cherry-picking anywhere I think.

**Comment by jasperjorna** [comment]
Is there a predicted release date for v0.64.3?

**Comment by mikehardy** [comment]
0.64.3 may never happen @jasperjorna - 0.65.0 is in RC2 phase and may release shortly, getting it out takes priority I believe

**Comment by grit96** [comment]
https://github.com/facebook/react-native/commit/2f477b1844cc168cf16b3fb600aec0b5ed558d3b allows jest 27 to work with react-native. It would be good to include if we do get a 0.64.3.

**Comment by mikehardy** [comment]
@grit96 this is even more important to note over on the 0.65.0 RC2 phase issue as it is not on the release-0.65 branch and that's where current activity is, can you note that over there? https://github.com/react-native-community/releases/issues/235

**Comment by zgordon01** [comment]
Any updates on if we are skipping 0.64.3 in favor of 0.65.0?

**Comment by kelset** [comment]
not much to update from my side sadly, been too busy with work stuff. The plan was for me to onboard another releaser and have them do 0.64.3 as their first "guided" release, but that feel through because this person was not able to get time at work to dedicate to this.

**Comment by taylorkline** [comment]
facebook/react-native@58444c74f5c18b74e88a6c1cd0f059fe434c1a21 applies cleanly and allows for optionally providing custom NSURLSession configurations. If a patch of 0.64 is made, this would be a nice inclusion for certain use cases.

**Comment by nextlevelbeard** [comment]
I believe the test automation community would highly appreciate these:
https://github.com/facebook/react-native/commit/efb359f3184ec26a21044b6145586cf1f5afc438 (https://github.com/facebook/react-native/pull/31865)
https://github.com/facebook/react-native/commit/91cac2028900cd18d17e70f9050cc125ed1eb12e (https://github.com/facebook/react-native/pull/31942)

Small but important fixes to the `testID` prop on Android builds.
It is currently (0.64) not working with certain important components like `TextInput` and `ScrollView`.

**Comment by mikehardy** [comment]
@nextlevelbeard at this point you are really going to want to post these on #244 since they do not appear to be on the release 65 branch or in a release yet. I doubt 64 will see more releases

**Comment by nextlevelbeard** [comment]
Thanks @mikehardy, it is done

**Comment by kelset** [comment]
Hey folks, we had a meeting last night and we decided that since 0.66 is already in the work (https://github.com/react-native-community/releases/issues/246), and that moving to 0.65 is fairly straightforward ([upgrade-helper](https://react-native-community.github.io/upgrade-helper/?from=0.64.2&to=0.65.1)) we won't be doing any more patches (unless something massive happens) for 0.64.
